projects/plugins/jetpack/_inc/client/writing/composing.jsx

```diff
@@ -198,6 +198,12 @@ export class Composing extends React.Component {
 										'jetpack'
 									) }
 								</span>
+								<span className="jp-form-setting-explanation">
```

Comment:
What do you think about adding a condition so this only shows when the toggle is active?

Reply:
6c75359b2c2c040f06d33d1519ea2d13081c410f